pageserver/src/tenant/timeline.rs

```diff
@@ -1136,6 +1168,8 @@ impl Timeline {
             }
         };
 
+        // TODO: update metrics for how
```

Comment:
Is it expected to be addressed before the merge?

Reply:
Looks like a partial sentence in my signature style. Metrics were added in separate PR, and we decided against pursuing more metrics.

Reply:
Cleaned up in [c4b2d3e](https://github.com/neondatabase/neon/pull/3809/commits/c4b2d3e40a340fec03c2d990f828ca700b9ebde3)